configure.ac

```diff
@@ -70,6 +70,7 @@ AC_ARG_ENABLE([direct],
 
 dnl Checks for programs
 AC_PROG_CC_C99
+AM_PROG_CC_C_O
```

Comment:
So `AM_PROG_CC_C_O` is obsolescent in Automake 1.14 and above because the `AC_PROG_CC` macro handles the same checks. In Automake 1.14 and above `AM_PROG_CC_C_O` is just a no-op. It isn't scheduled for removal until Automake 2.0. I don't know what the appropriate solution is.